Makefile

```
@@ -111,8 +111,8 @@ bin/api-generator:
 .PHONY: gen-api-commands
 gen-api-commands: bin/api-generator ## Generate api commands
```

The binary doesn't need to be generated anymore, correct? 

```suggestion
gen-api-commands: ## Generate api commands
```
This can also be removed
```
bin/api-generator:
	go build -o ./bin/api-generator ./tools/cmd/api-generator
```

---

Thanks, totally missed that!